README.md

```diff
@@ -1,2 +1,171 @@
-# budget-app
-Budget App is a Ruby on Rails mobile web application that allows users to manage their budget. Users can create a list of transactions associated with a category, so that they can see how much money they spent and on what. The app is also secure, with user registration and login to protect the data.
+# Budget App
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [Tengo Wallet](#Tengo-Wallet)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About the project ](#about-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Tengo Wallet <a name="about-project"></a>
+
+Budget App is a Ruby on Rails mobile web application that allows users to manage their budget. Users can create a list of transactions associated with a category, so that they can see how much money they spent and on what. The app is also secure, with user registration and login to protect the data.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li>Ruby</li>
+    <li>Rails</li>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Test-Driven Development
+- Class implementation with methods
+- Data Preserved in .json files
+- Unit tests created with Rspec
+- Backend PGSQL tables
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+git clone https://github.com/Elhambasir/budget-app.git
+
+```
+
+### Install
+
+To run the project, execute the folllowing in a terminal.
+
+```sh
+cd bugdet-app
+gem install
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+  rails server
+```
+
+
+### Run tests
+To run tests, run the following command:
+npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}"
+  rubocop .
+  rspec spec
+```
+### Deployment <a name="deployment"></a>
+
+- [Live](comming soon)

```

Comment:
- [ ] As requested by the previous reviewer, kindly deploy the app and add the live link. Follow this [resource](https://docs.google.com/document/d/1WiY5KoY6IS1DiN0tX5LiuP3x7entO6dmJlnGSGiibQA/edit?pli=1). If you follow these steps you will make some commits to change your application and fix the issue